skt/kernelbuilder.py

```diff
@@ -73,7 +73,15 @@ def clean_kernel_source(self):
 
     def prepare_kernel_config(self):
         """Prepare the kernel config for the compile."""
-        shutil.copyfile(self.basecfg, "%s/.config" % self.source_dir)
+        if self.cfgtype == 'rh-configs':
+            # Build Red Hat configs and copy the correct one into place
+            self.make_redhat_config()
+        else:
+            # Copy the existing config file into place
+            shutil.copyfile(self.basecfg, "%s/.config" % self.source_dir)
+            args = self.make_argv_base + [self.cfgtype]
+            logging.info("prepare config: %s", args)
+            subprocess.check_call(args)
```

Comment:
It would be nice to the reviewers to have a little explanation of what it means to handle rh-configs properly, in the commit message.

Not necessary for an ACK from me, but I would be glad to see that in the future.

Reply:
Done.